layers/+tools/transmission/README.org

```diff
@@ -63,52 +64,68 @@ Add =(transmission :variables transmission-auto-refresh-all t)= to
 
 ** =transmission-mode=
 
-| Key binding | Description                                                   |
-|-------------+---------------------------------------------------------------|
-| ~SPC m g t~ | Open a `transmission-files-mode' buffer for torrent at point. |
-| ~SPC m a t~ | Add torrent by filename, URL, magnet link, or info hash.      |
-| ~SPC m s d~ | Set global download speed limit in kB/s.                      |
-| ~SPC m g p~ | Open a =transmission-peers-mode= buffer for torrent at point. |
-| ~SPC m g i~ | Open a =transmission-info-mode= buffer for torrent at point.  |
-| ~SPC m s l~ | Set global seed ratio limit.                                  |
-| ~SPC m m~   | Move torrent at point or in region to a new location.         |
-| ~SPC m r~   | Prompt to remove torrent at point or torrents in region.      |
-| ~SPC m t s~ | Toggle torrent between started and stopped.                   |
-| ~SPC m a a~ | Add announce URLs to torrent or torrents.                     |
-| ~SPC m s u~ | Set global upload speed limit in kB/s.                        |
-| ~SPC m v~   | Verify torrent at point or in region.                         |
-| ~SPC m q~   | Quit and bury the buffer.                                     |
-| ~SPC m s b~ | Set bandwidth priority of torrent(s) at point or in region.   |
+| Key binding | Description                                                       |
+|-------------+-------------------------------------------------------------------|
+| ~H~         | Move torrent to the top of the queue.                             |
+| ~J~         | Move torrent down the queue by one.                               |
+| ~K~         | Move torrent up the queue by one.                                 |
+| ~L~         | Move torrent to the bottom of the queue.                          |
+| ~SPC m m~   | Move torrent at point or in region to a new location.             |
+| ~SPC m q~   | Quit and bury the buffer.                                         |
+| ~SPC m v~   | Verify torrent at point or in region.                             |
+| ~SPC m D~   | Prompt to remove and delete torrent at poit or torrens in region. |
```

Comment:
description of remove and delete needs to be improved
the difference between them is `transmission-remove` only remove the torrent by default, while `transmission-delete` in addition remove local data. (`transmission-remove` prompts whether to remove local data)

See https://github.com/transmission/transmission/blob/8566df069899ce8923463cadeb0ff66d4544991a/extras/rpc-spec.txt#L439

Reply:
I'll work on the description for these 2. The reference you provided is perfect! I never thought of checking the source for Transmission.

The reason why I thought of changing the binding to "R" and "D" respectively was because they're both destructive executions (removing data from the client/disk). Both `transmission-remove` and `transmission-delete` prompt the user for confirmation of the removal (and deletion) before carrying out the process.

Reply:
I've made the descriptions a little clearer. I've left the bindings as "R" and "D" for the purpose of having `transmission-move` and `transmisison-mark` not clash with each other.

    | `SPC m D`    | Delete from disk and remove torrent(s) at point or in region.     |
    | `SPC m R`    | Remove torrent(s) at point or in region.                          |